.gitignore

```
@@ -1,6 +1,7 @@
 **/__pycache__
 .coverage 
 .idea
+.vscode
```

Optional: I've found `.vscode/extensions.json` and `.vscode/tasks.json` to be useful to check into source control. Consider:

```suggestion
.vscode/settings.json
```